docs/pages/contributing/documentation.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Contributing to Teleport's Documentation
+description: Resources for contributing to Teleport's documentation
+layout: tocless-doc
+---
+
+Documentation is essential for learning about a product or service. Even for internal contributors, reference documentation is often the ultimate source of truth and the final arbiter of disputes and disagreements. And for decision makers at organizations evaluating solutions, documentation is often a key resource.
```

Comment:
nit:

I typically prefer to hard wrap lines at a reasonable point rather than have really long lines. It's easier to read in an editor, and also easier to review and comment on individual lines rather than paragraphs.

There's lots of editor plugins that can do this automatically. I prefer `reflow-markdown` for vs-code.